infra/create-app/main.tf

```diff
@@ -75,3 +75,12 @@ module "sns" {
   project_name = local.project_name
 }
 
+module "cloudfront" {
+  source       = "../modules/cloudfront"
+  environment  = local.environment
+  project_name = local.project_name
+  providers = {
+    aws           = aws
+    aws.us-east-1 = aws.useast1
```

Comment:
What happens here then? Noticed you have the provider set on the module aswell

Reply:
It seems that you have to define providers at the root level and pass them down to modules, if you want to use a non-default provider in the module. This bit is how the providers are passed to the module (since we need to create stuff in us-east-1 and eu-west-1 in the same module, we need both). Then in the module itself you define the required_providers block to tell it what to expect from the root level.